master/getting-started/kubernetes/upgrade/downgrade.md

```diff
@@ -25,26 +46,15 @@ follow the steps here to downgrade.
    kubectl rollout undo deployment/calico-kube-controllers -n kube-system
    ```
 
-1. Watch the status of the downgrade as follows.
+1. Watch the status of the downgrade as follows. When it reports complete and
+   successful, {{site.prodname}} is downgraded to the previous version.
 
    ```
-   watch kubectl get pods -n kube-system
+   kubectl rollout status deployment/calico-kube-controllers -n kube-system
+   kubectl rollout status ds/calico-node -n kube-system
    ```
-   
-   Verify that the status of all {{site.prodname}} pods indicate `Running`.
 
-   ```
-   calico-kube-controllers-6d4b9d6b5b-wlkfj   1/1       Running   0          3m
-   calico-node-hvvg8                          2/2       Running   0          3m
-   calico-node-vm8kh                          2/2       Running   0          3m
-   calico-node-w92wk                          2/2       Running   0          3m
-   ```
-
-## Downgrading a custom installation
-
-_Docs for this coming soon!_
-
-## Aborting the upgrade
+## Aborting the upgrade when using the etcd datastore
 
 If you have not upgraded the {{site.prodname}} components then follow
```

Comment:
This is a little confusing after being directed here by the Downgrade instructions. I'll rework or remove it.